murfi-rt-PyProject/scripts/feedback.sh

```diff
@@ -100,7 +134,10 @@ then
 clear
     echo "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
     echo "ready to receive rtdmn feedback scan"
-    singularity exec /home/auerbachlinux/singularity-images/murfi2.sif murfi -f $subj_dir/xml/rtdmn.xml
+    export MURFI_SUBJECTS_DIR="${absolute_path}/subjects/"
+    export MURFI_SUBJECT_NAME=$subj 
+
+    singularity exec --bind home/rt:/home/rt murfi2.sif murfi -f $subj_dir_absolute/xml/rtdmn.xml
```

Comment:
Here, I've bound a path to the `fsl_reg_subj_epi.sh` script, which I believe is supposed to be used to register all the ROI masks to the first volume of each feedback run. So far, I haven't gotten this to actually run yet though, as FSL is not installed in the`murfi2.sif` singularity image we have

Reply:
Since Isaac already tinkered with the Docker, would it make sense to try and add fsl to it too so it also gets into the singularity image?